src/fabric/src/fabric_util.erl

```diff
@@ -152,6 +152,13 @@ all_docs_timeout() ->
 attachments_timeout() ->
     timeout("attachments", "600000").
 
+global_view_timeout(Args) ->
+    PartitionQuery = couch_mrview_util:get_extra(Args, partition, false),
+    case PartitionQuery of
+        false -> timeout("global_view_timeout", "infinity");
```

Comment:
You're doubling up the _timeout suffix cause the timeout/2 call appends that as well. Also global_view_timeout should probably be shortened to just view_timeout because you're looking at global vs partitioned. Also global_partition is a bit of a misnomer and should just be partition_view_timeout.

Also I think the SOP is to add commented out config options with their default values in default.ini. If the other timeout config options aren't already present I'd add all five settings in a separate commit on this PR.

Reply:
I've removed the `global_` part for the names. The reason I added that is that rexi has two timeouts. A timeout for how long to wait for each response from a node and then an overall global timeout for the node to send all responses. The view and partition_view are for the second situation where as the `all_docs` and `request` timeouts are fo the individual requests.